source/fundamentals/aggregation/aggregation-expression-operations.txt

```diff
@@ -333,21 +334,30 @@ this aggregation:
 .. code-block:: java
    :copyable: true
 
-   public static MqlNumber classAvg(MqlArray<MqlDocument> u) {
-       var sum = u.sum(x -> x.getInteger("finalGrade"));
-       var avg = sum.divide(u.size());
+   public static MqlNumber gradeAverage(MqlArray<MqlDocument> students, String fieldName) {
+       var sum = students.sum(student -> student.getInteger(fieldName));
+       var avg = sum.divide(students.size());
        return avg;
    }
-   
-   public static MqlString evalAvg(MqlNumber v) {
-       MqlNumber[] b = {of(70), of(85)};
-       var msg = v.switchOn(on -> on
-               .lte(b[0], m -> of("Needs improvement"))
-               .lte(b[1], m -> of("Meets expectations"))
+
+   public static MqlString evaluate(MqlNumber avg, MqlNumber[] benchmarks) {
```

Comment:
```suggestion
   public static MqlString evaluate(MqlNumber grade, MqlNumber toMeet, MqlNumber toExceed) {
```
If we need multiple values, we should use a list, or the `MqlNumber...` syntax, but here we are constrained to exactly two values because of the 3 resulting evaluations. Alternative naming might be low/high, cutoff1/cutoff2, first/second, or something else.

The grade might not be an average.